src/seismometer/_api.py

```diff
@@ -23,7 +31,8 @@
 from .data.filter import FilterRule
 from .data.timeseries import create_metric_timeseries
 from .html import template
-from .report.auditing import display_fairness_audit
+from .html.iframe import load_as_iframe
+from .report.auditing import fairness_audit_to_html
 from .report.profiling import ComparisonReportWrapper, SingleReportWrapper
 from .seismogram import Seismogram
 
```

Comment:
Line 76: This refers to the typehint for `exclude_cols`: should be Optional[list[str]] = None

Maybe this is fine, but it would be inconsistent with typehints in `thresholds.py`

Reply:
FINC